bluedonkey.py

```diff
@@ -57,26 +65,35 @@ def init_filter():
 
 class dummy_car_control():
     def __init__(self):
+        ## Commented per jkridner's advice
         import car_control
         self.c = car_control.car_control()
+        
+        #Output for the status
+        self.status_port = 3004
+        self.status_out = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        self.status_out.setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
+        self.status_out.connect(("", self.status_port))
+        self.status_file = self.status_out.makefile('w', buffering=None)
+        #pass
 
     def tick(self):
         self.c.tick()
         return
 
     def update(self, line, threshold):
-        (paused, throttle, steering, fps) = self.c.update(line)
-        if paused:
-            print("P ", end="", flush=False)
+        (self.paused, self.throttle, self.steering, self.fps) = self.c.update(line)
+        if self.paused:
+            print("P ", end="", flush=False, file=self.status_file)
         else:
-            print("  ", end="", flush=False)
+            print("  ", end="", flush=False, file=self.status_file)
         if line:
-            print("%03d %03d " % (line[2], line[3]), end="", flush=False)
+            print("%03d %03d " % (line[2], line[3]), end="", flush=False, file=self.status_file)
         else:
-            print("No line ", end="", flush=False)
-        print("%06.2f %06.2f" % (throttle, steering), end="", flush=False)
-        print(" %04.1f" % (fps), end="", flush=False)
-        print(" %03d" % (threshold), end="", flush=False)
-        print("\r", end="", flush=True)
+            print("No line ", end="", flush=False, file=self.status_file)
+        print("%06.2f %06.2f" % (self.throttle, self.steering), end="", flush=False, file=self.status_file)
+        print(" %04.1f" % (self.fps), end="", flush=False, file=self.status_file)
+        print(" %03d" % (threshold), end=" ", flush=False, file=self.status_file)
+        print("Live Tracker: ",randint(0,100), flush=False, file=self.status_file)
```

Comment:
What does this random number do?

Reply:
The random number was a temporary debug measure to allow us to verify that the dashboard was truly updating, and allowed us to troubleshoot our loading of the test config file. The random number will be removed for the final submission.